src/Facets/SynapseBridgeFacet.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { ISynapseRouter } from "../Interfaces/ISynapseRouter.sol";
+import { LibAsset, IERC20 } from "../Libraries/LibAsset.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { SwapperV2, LibSwap } from "../Helpers/SwapperV2.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+
+/// @title SynapseBridge Facet
+/// @author LI.FI (https://li.fi)
+/// @notice Provides functionality for bridging through SynapseBridge
+contract SynapseBridgeFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    /// Storage ///
+
+    address internal constant NETH_ADDRESS =
+        0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+
+    /// @notice The contract address of the SynapseRouter on the source chain.
+    ISynapseRouter private immutable synapseRouter;
```

Comment:
We can make this public just so it's easier to verify after deployment

Reply:
updated it.
https://github.com/lifinance/contracts/pull/234/commits/d4f9c11982b08173f86a9d9a4b04ebb67df9b7ef